examples/gemm_fp4/example_fusedmoe_a8w4_sm120.py

```
@@ -0,0 +1,178 @@
+    block_token=128,
+    block_hidden=128,
+    block_expert=128,
+    threads=128,
+    num_stages=1,
+):
+    """Single shared expert: gate_up GEMM -> SiLU*up -> down GEMM."""
+    scale = 1.44269504  # log2(e) for fast SiLU
+
+    @T.prim_func
+    def main(
+        input: T.Tensor((num_tokens, d_hidden), "float8_e4m3fn"),
```

Ruff (0.15.12)

[error] Function argument `input` is shadowing a Python builtin

(A002)